docs/sketch-project-file.md

```diff
@@ -115,3 +115,24 @@ When using the profile-based build, the globally installed platforms and librari
 not be used in any way. In other words, the build is isolated from the system and will rely only on the resources
 specified in the profile: this will ensure that the build is portable and reproducible independently from the platforms
 and libraries installed in the system.
+
+## Default flags for Arduino CLI usage
+
+The sketch project file may be used to set the default value for some command line flags of the Arduino CLI, in
+particular:
+
+- The `default_fqbn` directive sets the default value for the `--fqbn` flag
+- The `default_port` directive sets the default value for the `--port` flag
+- The `default_protocol` directive sets the default value for the `--protocol` flag
```

Comment:
```suggestion
- The `default_fqbn` key sets the default value for the `--fqbn` flag
- The `default_port` key  sets the default value for the `--port` flag
- The `default_protocol` key sets the default value for the `--protocol` flag
```

I think "key" is the correct terminology here.

References:

- https://yaml.org/spec/1.2.2/ext/glossary/#:~:text=M-,Mapping,-A%20kind%20of
- https://www.yaml.info/learn/index.html#:~:text=items%20and%20more.-,Mapping,-The%20most%20common